### ISSUE TITLE
Direct rooms are not recognised as a DM

### ISSUE THREAD

**Issue by freswa** [ulyssa/iamb#311]
When I open `:dms`, the list is empty: `No direct messages yet!`.
Though, other clients show me my DMs just fine. Iamb though, shows them as usual rooms to me.

**Comment by burdges** [comment]
Yup, direct messages show up in :rooms while :dms stays empty.  

According to https://iamb.chat/rooms/browsing.html#browsing-direct-messages :rooms should not show the direct messages, only :chats should show both together.

**Comment by VAWVAW** [comment]
You can manually mark a room as a dm with `:room dm set`. After that it should show up under `:dms`. 

There is probably a problem with the room creation that doesn't mark the room as a DM.
